experimental/language-generation/csharp_dotnetcore/05.multi-turn-prompt/Resources/UserProfileDialog.LG

```diff
@@ -8,10 +8,10 @@
 - Please enter your name.
 
 # AckName
-- Thanks, @{CapitalizeFirstLetter(Result)}.
+- Thanks, ${CapitalizeFirstLetter(Result)}.
 
 # CapitalizeFirstLetter (arg) 
-- @{concat(toUpper(substring(arg, 0, 1)), substring(arg, 1))}
+- ${concat(toUpper(substring(arg, 0, 1)), substring(arg, 1))}
 
```

Comment:
this seems like it should be in standard functions.

Reply:
Will add to R9 backlog to add a function.